components/layout/app-bar.jsx

```diff
@@ -25,8 +26,14 @@ const DashboardAppBar = ({ children, className, ...restProps }) => {
       </AppBar.Brand>
 
       {children}
+      <AppBar.Item>
+        {`${store.user.givenName} ${store.user.familyName} `}|{' '}
```

Comment:
Could you pleas add here some `span` with class that would hide on mobile screens?